src/jest-puppeteer.config.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * For a detailed explanation of configuration properties, visit:
+ * https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#puppeteerlaunchoptions
+ */
+
+const { CI, E2E_DEBUG, PUPPETEER_HEADLESS, PUPPETEER_SLOWMO } = process.env;
+let executablePath = '';
+let dumpio = false;
+if ( ! CI ) {
+    executablePath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
```

Comment:
This will work only for MacOS, so it might be a good idea to add a way to override the path somehow. Not something that needs to be fixed in this PR though.

Reply:
Good idea, I'll update that later.